frontend/src/views/auth/NavigationAuth.vue

```diff
@@ -0,0 +1,74 @@
+<template>
+  <v-app-bar
+    v-if="$vuetify.breakpoint.mdAndUp"
+    app
+    clipped-left
+    color="blue-grey darken-4"
+    dark
+  >
+    <logo text />
+    <v-spacer />
+    <v-toolbar-items>
+      <v-btn :href="newsLink" target="_blank" text>
```

Comment:
On the unauthenticated views, on mobile there is no way to get to the news, correct? Is that intentional?

Reply:
Correct. I did not find (and also probably look for) a good solution.